src/core/stdc/stdio.d

```diff
@@ -643,10 +643,7 @@ else version (DragonFlyBSD)
 }
 else version (Solaris)
 {
-    import core.stdc.wchar_ : __mbstate_t;
-
-    ///
-    alias mbstate_t = __mbstate_t;
```

Comment:
Do we have a policy on the C bindings and breaking headers?

Reply:
It was already broken in #2473, this makes it consistent with the rest.

Reply:
It would be nice to have more strict documentation in each binding module that tells the porter what do define here.

Currently, people just put whatever they want wherever they want / require it.

Reply:
Yeah, that would be nice.

Reply:
This is a good example:
https://github.com/dlang/druntime/blob/3d4d4376cebaeb9d624834d334d3c287e010dccf/src/core/sys/posix/fcntl.d#L53-L102